GirafIntegrationTest/tests/test_account_controller.py

```diff
@@ -0,0 +1,355 @@
+from requests import get, post, put, delete
+import time
+from testlib import order, BASE_URL, auth, GIRAFTestCase
+
+graatand_token = ''
+graatand_id = ''
+gunnar_token = ''
+gunnar_reset_token = ''
+gunnar_username = ''
+gunnar_id = ''
+grundenberger_token = ''
+grundenberger_id = ''
+grundenberger_username = ''
+tobias_token = ''
```

Comment:
These are very specific. One may not know what **gunnar** or **grudenberger** is. 

I believe there are a difference of the tokens, thus you could find suitable names for them. If the `graatand` user is a **citizen**, then call it `citizen_user_token` or something like that. 

With this refactoring, you can name your tests better and more descriptive:
```
def test_cannot_login_to_deleted_citizen_account(self)
def test_citizens_cannot_delete_guardian_account(self)
def test_can_login_as_guardian(self)
...
```

And so forth.